templates/core/features/recovery.html

```diff
@@ -1,141 +1,174 @@
+                 alt=""
+                 width="300" />
+          </div>
+        </div>
+      </div>
     </div>
-    <div class="col-4 u-vertically-center u-hide--medium u-hide--small u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/7de21966-snapshot.svg",
-        alt="",
-        width="150",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+    <div class="row">
+      <div class="col-9 col-start-large-4 col-medium-5 col-start-medium-2">
+        <hr class="p-rule is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">Snapshots></h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Multiple snapshots for the same device can be backed up in the recovery system. These snapshots reflect both configuration settings and the collection of snaps installed on the system. Device operators can create, name and save such snapshots in the recovery system. This enables a swift recovery of a desired system state when needed. Recovery can happen manually through a dedicated UI or remotely via an API call.
+            </p>
+            <img src="https://assets.ubuntu.com/v1/69d7736d-snapshots.png"
+                 alt=""
+                 width="300" />
+          </div>
+        </div>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-strip is-deep">
-  <div class="row u-equal-height">
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-            alt="",
-            width="281",
-            height="200",
-            hi_def=True,
-            loading="lazy"
-            ) | safe
-          }}
-    </div>
-    <div class="col-7">
-      <h2>Low touch recovery for your devices</h2>
-      <p>Get in touch with an Ubuntu security expert to discuss the advanced security requirements of your application.</p>
-      <p>
-        <a href="/core/contact-us?product=core-recovery" class="p-button--positive js-invoke-modal">Get in touch</a>
-      </p>
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Low touch recovery <br /> for your devices</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted">
+            <img class="p-image-container__image"
+                 src="https://assets.ubuntu.com/v1/683e6225-Image.png"
+                 alt="" />
+          </div>
+        </div>
+        <p>Get in touch with an Ubuntu security expert to discuss the advanced security requirements of your application.</p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/core/contact-us?product=core-recovery"
```

Comment:
```suggestion
          <a href="/core/contact-us"
```
js-invoke-modal class will invoke the required form here but if that doesn't happen then it should direct to contact page.